<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoutubeWindow</class>
 <widget class="QDialog" name="YoutubeWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Search</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-internet">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelSearch">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="textSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearch">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelResults">
     <property name="text">
      <string>Results:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listResults">
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSearchRelated">
       <property name="text">
        <string>Search Related</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelState">
       <property name="text">
        <string>&lt;state&gt;</string>
       </property>
       <property name="indent">
        <number>-13</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPreferences">
       <property name="text">
        <string>Preferences</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonClose">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textSearch</tabstop>
  <tabstop>buttonSearch</tabstop>
  <tabstop>listResults</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>listResults</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>buttonAdd</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textSearch</sender>
   <signal>returnPressed()</signal>
   <receiver>buttonSearch</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClose</sender>
   <signal>rejected()</signal>
   <receiver>YoutubeWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
